### ISSUE TITLE
Let people know about the project (1 minute task)

### ISSUE THREAD

**Issue by mrjohnc** [mrjohnc/Map-of-the-open-movement#2]
Let people know about the project (1 minute task): [Promote the project on Twitter](https://twitter.com/mrjohnc/status/993943192503574528)

**Comment by nyakwakah** [comment]
Hi! My name is Hildah, I live in Nairobi, Kenya and I am very interested in this project. I'd love to help out as much as possible, starting maybe from here? I assume it's the starting point. Could you post the link here so that I share it on mine and my organization's Twitter? Thanks :smiley_cat:

**Comment by mrjohnc** [comment]
@nyakwakah thanks, sorry, forgot to add the link here, now fixed

**Comment by mrjohnc** [comment]
@nyakwakah Yes please start with retweet and then have a look at readme https://github.com/mrjohnc/Map-of-the-open-movement/blob/master/README.md

Thanks
